docs/SETUP.md

```diff
@@ -70,6 +70,25 @@ We use Postgres around these parts. Installation will differ based on your OS.
   * Other [linux instructions](https://www.postgresql.org/download/) (server and developer libraries needed)
 * Run the command `rails db:create && rails db:migrate` to set up the tables
 
+* Configure Postgres
+ * Setup your user; the suggested password is `postgres` if you only use this machine for development purposes
+   ```
+   sudo -u postgres createuser -s `whoami`
+   ```
+ * Set your env variables:
+   * Mac:
+     ```
+     echo "export POSTGRES_USER=`whoami`" >>~/.bashrc
```

Comment:
I wonder if `dotenv` might save us some kinda particular/confusing shell mucking by letting us just stick this in an `.env` file in the root of the project? https://github.com/bkeepers/dotenv I will defer to you whether or not it's worth the dependency add.

Reply:
I personally like dotenv a lot.